e2e/config/config.go

```diff
@@ -1,50 +1,40 @@
 package config
 
 import (
-	"os"
-	"strings"
 	"time"
+
+	"github.com/caarlos0/env/v11"
+	"github.com/joho/godotenv"
 )
 
 var (
-	AirgapNSGName                 = "abe2e-airgap-securityGroup"
-	Azure                         = MustNewAzureClient(SubscriptionID)
-	BuildID                       = lookupEnvWithDefaultString(os.Getenv("BUILD_ID"), "local")
-	DefaultSubnetName             = "aks-subnet"
-	IgnoreScenariosWithMissingVHD = lookupEnvWithDefaultBool("IGNORE_SCENARIOS_WITH_MISSING_VHD", false)
-	KeepVMSS                      = lookupEnvWithDefaultBool("KEEP_VMSS", false)
-	Location                      = lookupEnvWithDefaultString("LOCATION", "westus3")
-	ResourceGroupName             = "abe2e-" + Location
-	SIGVersionTagName             = lookupEnvWithDefaultString("SIG_VERSION_TAG_NAME", "branch") // ADO tags every SIG image version with `branch` tag. By specifying `branch=refs/heads/master` we load latest image version from the master branch.
-	SIGVersionTagValue            = lookupEnvWithDefaultString("SIG_VERSION_TAG_VALUE", "refs/heads/master")
-	SkipTestsWithSKUCapacityIssue = lookupEnvWithDefaultBool("SKIP_TESTS_WITH_SKU_CAPACITY_ISSUE", false)
-	SubscriptionID                = lookupEnvWithDefaultString("SUBSCRIPTION_ID", "8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8")
-	TagsToRun                     = os.Getenv("TAGS_TO_RUN")
-	TagsToSkip                    = os.Getenv("TAGS_TO_SKIP")
-	TestTimeout                   = lookupEnvWithDefaultDuration("TEST_TIMEOUT", 18*time.Minute)
-	E2ELoggingDir                 = lookupEnvWithDefaultString("LOGGING_DIR", "scenario-logs")
+	Cfg               = mustLoadConfig()
```

Comment:
nit: naming seems redundant, considering renaming the package to `suite` so we can do `suite.Config` instead of `config.Cfg`?

can also rename the struct to `Configuration` to accomplish that

Reply:
I couldn't figure out a better name. But I feel like `config` is a better package name than `suite`.